src/codecs/plos/__file_snapshots__/pbio-3000349.yaml

```diff
@@ -1,4 +1,7 @@
 type: Article
+about:
+  - type: DefinedTerm
+    name: Research Article
```

Comment:
For these PLOS articles, it seems the `genre: Research Article` has become an `about`?

Reply:
Hi @nokome, I think yes he has to convert.
From all members of eLife team, we thank you for the collaboration and we hope to do that again in the future.